pkgs/os-specific/linux/alsa-project/alsa-utils/default.nix

```
@@ -30,6 +43,7 @@ stdenv.mkDerivation rec {
   postFixup = ''
     mv $out/bin/alsa-info.sh $out/bin/alsa-info
     wrapProgram $out/bin/alsa-info --prefix PATH : "${lib.makeBinPath [ which pciutils ]}"
+    wrapProgram $out/bin/aplay --set ALSA_PLUGIN_DIR ${plugin-dir}
```

```suggestion
    wrapProgram $out/bin/aplay --set-default ALSA_PLUGIN_DIR ${plugin-dir}
```
just in case to make this overwritable